packages/react-components/icons/BackChevron.v2.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import Svg, { Path } from 'svgs'
+
+export interface Props {
+  height: number
+  color: string
+}
+
+export default function BackChevron({ color, height }: Props) {
```

Comment:
Setting default static props here like we do for some other icons may be a nice addition